en/flight_controller/kakuteh7mini.md

```diff
@@ -0,0 +1,141 @@
+# Holybro Kakute H7 mini
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://shop.holybro.com/) for hardware support or compliance issues.
+:::
+
+The [Holybro Kakute H7 mini](http://www.holybro.com/product/kakute-h7-mini/) flight controller is full of features including integrated Bluetooth, HD camera plug, dual plug-and-play 4in1 ESC ports, 9V VTX ON/OFF Pit Switch, barometer, OSD, 6x UARTs, 128MB Flash for Logging (not supported with PX4 yet), 5V and 9V BEC, and bigger soldering pad with easy layout and much more.
```

Comment:
Yes it has lots of features, but is it intended for a particular use case? I.e. racers etc? i.e. what would you use mini vs h7 for?

This would be better worded as something like: "This is a flight controller intended for racers. It has many features, including:"

Reply:
That would be good. @vincentpoont2

Reply:
@vincentpoont2 @julianoes Progress ?

Reply:
@hamishwillee I will review this today.

Reply:
It is intended for Racer, but I don't think a lot of racer use PX4. I think it is good for people wanting to stay light weight and keep the the frame small.